pkgs/top-level/all-packages.nix

```diff
@@ -8580,6 +8580,9 @@ in
   apacheKafka_2_1 = callPackage ../servers/apache-kafka { majorVersion = "2.1"; };
   apacheKafka_2_2 = callPackage ../servers/apache-kafka { majorVersion = "2.2"; };
 
+  apacheFlume = apacheFlume_1_9_0;
+  apacheFlume_1_9_0 = callPackage ../servers/apache-flume { majorVersion = "1.9.0"; };
```

Comment:
Do you think there will be reasons to keep multiple versions of flume?
It does not seem like there are multiple maintained at the same time.
If that's not the case, it'd be better not to have to maintain multiple versions for us.